services/horizon/internal/ingest/processors/operations_processor.go

```diff
@@ -688,11 +697,18 @@ func (operation *transactionOperationWrapper) Details() (map[string]interface{},
 			}
 			details["parameters"] = params
 
-			if balanceChanges, err := operation.parseAssetBalanceChangesFromContractEvents(); err != nil {
-				return nil, err
+			var balanceChanges []map[string]interface{}
+			var parseErr error
+			if !operation.skipSorobanDetails {
```

Comment:
nit suggestion: double-negatives can be annoying to read - can we invert the conditionals? i.e.
```suggestion
			if operation.skipSorobanDetails { // elide
```

Reply:
yep, done - https://github.com/stellar/go/pull/5176/commits/4765d5242e69008bc977377cbb7be80d017217c8